### PR TITLE
Added sepia tone

### PR DESCRIPTION
**Pull request by mateuszz0000**
### **Describe your change:**
Added sepia tone in digital_image_processing.

* [X] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [X] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by poyea** [comment]
Looks good from the result.

File: digital_image_processing/sepia.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    def to_grayscale(blue, green, red):
+        """
+        Helper function to create pixel's greyscale representation
+        Src: https://pl.wikipedia.org/wiki/YUV
+        """
+        return 0.2126 * red + 0.587 * green + 0.114 * blue
+
+    def normalize(value):
+        """ Helper function to normalize R/G/B value -> return 255 if value > 255"""
+        return value if value <= 255 else 255
```
Comment (cclauss) [inline digital_image_processing/sepia.py:21]:
```suggestion
        return min(value, 255)
```
https://docs.python.org/3/library/functions.html#min

File: digital_image_processing/sepia.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    img = imread("image_data/lena.jpg", 1)
+    img1 = imread("image_data/lena.jpg", 1)
+    img2 = imread("image_data/lena.jpg", 1)
+    img3 = imread("image_data/lena.jpg", 1)
+
+    # convert with sepia with different factor's value
+    sepia_10 = make_sepia(img, 10)
+    sepia_20 = make_sepia(img1, 20)
+    sepia_30 = make_sepia(img2, 30)
+    sepia_40 = make_sepia(img3, 40)
```
Comment (cclauss) [inline digital_image_processing/sepia.py:46]:
```suggestion
    images = {percentage: imread("image_data/lena.jpg", 1) for percentage in (10, 20, 30, 40)}
    for i, percentage, img in enumerate(images.items()):
        images[i] = make_sepia(img, percentage)
    for i, percentage, img in enumerate(images.items()):
        imshow(f"Original image with sepia (factor: {percentage})", img)
```

**Comment by mateuszz0000** [comment]
Finally fixed :)

**Comment by cclauss** [comment]
Nice contribution!  Thx

**Pull request merged**